debian/rules

```diff
@@ -44,7 +44,7 @@ override_dh_auto_configure:
 	meson --buildtype=plain --prefix=/usr --sysconfdir=/etc --libdir=lib/$(DEB_HOST_MULTIARCH) --bindir=lib/nnstreamer/bin --includedir=include \
 	-Dtf-support=$(enable_tf) -Dtflite-support=enabled -Dpytorch-support=enabled -Dcaffe2-support=enabled \
 	-Dpython2-support=enabled -Dpython3-support=enabled -Denable-capi=true -Denable-edgetpu=true -Denable-tizen=false \
-	-Denable-openvino=true ${BUILDDIR}
+	-Denable-openvino=true -Dmxnet-support=enabled ${BUILDDIR}
```

Comment:
Don't enable this here. `meson.build` will detect its dependency automatically.

Reply:
Do you mean that the distributed Ubuntu PPA package won't explicitly add the MXNet support?

Reply:
Regardless of the PPA's MXNet support...
If you add `-Dmxnet-support=enabled` here, meson will try to build mxnet unconditionally, which results in CI failures.

I think `auto` is enough and please check https://mesonbuild.com/Build-options.html#features

Reply:
I understand why pr-postbuild-build-ubuntu currently fails. But AFAIK about the currently build/CI system, without `-Dmxnet-support=enabled` the code in the PR is left untested by the CI, which practically means that it is left unsupported. Is there another way to ensure my code works by CI?

Reply:
In fact, even without `Dmxnet-support=enabled`, your PR can be tested if the CI server can detect the mxnet package installed.

So, the problem is whether the CI server has the related dependency, not this option. That's why I said 'don't enable this here' in `debian/rules`.

In short, currently, your PR cannot be checked in CI server until someone provides the PPA. Even after someone does, `-Dmxnet-support=enabled` is not necessary.

Reply:
I see, I will remove it in the next revision (next rebase).

I would like to hear about the project's policy (or process) regarding adding a new backend. Because what you explained is exactly what I expected. When I submitted the PR, I thought this PR has to wait until someone decides to include MXNet package so that it can be tested by the CI server before it is merged. Do you think the PR would be processed first and then your team would decide whether to include MXNet later? (or independently to the status of this PR?)

Reply:
I'm not sure but I think that your PR can be review and merged first if you surely tested it in local env.
Actually, unless it affects the overall build process for public packages, I think it's okay.

Reply:
> your PR can be reviewed and merged first if you surely tested it in your local env.

I understood now. That's all what I would like to know :)

Reply:
To add mxnet in Ubuntu/PPA, you need to update the dependency debian/control and make sure that MXNET library exists in Ubuntu.

If MXNet doesn't exist in Ubuntu (or our PPA), you cannot add it to debian/control and thus, you cannot set it "enabled" in debian/rules.

Reply:
Thanks for more details about packaging. I will focus on the coding stuff in this PR then.